packages/fluent-theme/src/fluent/styles/SpinButton.styles.ts

```diff
@@ -0,0 +1,48 @@
+import { fluentBorderRadius } from './styleConstants';
+import { NeutralColors } from '../FluentColors';
+
+const buttonStyles = {
+  color: NeutralColors.gray130,
+  width: 23,
+  selectors: {
+    ':hover': {
+      backgroundColor: NeutralColors.gray20,
+      color: NeutralColors.gray130
+    },
+    ':active': {
+      backgroundColor: NeutralColors.gray30,
+      color: NeutralColors.gray130
+    },
+    '.ms-Button-icon': {
+      fontSize: 8
+    }
+  }
+};
+
+export const SpinButtonStyles = {
+  spinButtonWrapper: {
+    borderRadius: fluentBorderRadius,
+    borderColor: NeutralColors.gray80
+  },
+  spinButtonWrapperHovered: {
+    borderColor: NeutralColors.gray160
+  },
+  input: {
+    padding: '0 8px',
+    width: 'calc(100% - 23px)', // -23px because buttons width changed
```

Comment:
Related to above--would this be necessary if we don't change the width? `calc` can be expensive and it's best to avoid them if possible.

Reply:
This one is odd too, as it's also present in the default styles and decided not to risk breaking an edge case this calc was possibly introduced in the first place.

Personally, I consider the default styles badly need some refactoring for a couple reasons:
1. There almost no className hooks for Sass overrides. Here how the tree looks now:
<img width="437" alt="screen shot 2018-10-30 at 8 50 10 am" src="https://user-images.githubusercontent.com/29514833/47731331-1ccc9080-dc21-11e8-9a6e-e25c2ad6092f.png">

2. The styles are trying to support an older implementation with `getClassNames` prop by mixing the logic with the `styles` prop and not offering any `styleProps` to conditionally alter the styles (just a bunch of different classNames for all the possible states).

3. The `calc` function is most likely needed because the positioning of some elements (specifically the input field with the up and down buttons) is using `float`. Would be nice to have it converted to `flex` so it does it for you.

4. There is some misalignment of the label being slightly uncentered relative to the input field text, probably caused by `floats` again.
<img width="309" alt="screen shot 2018-10-30 at 9 00 36 am" src="https://user-images.githubusercontent.com/29514833/47731998-8ef1a500-dc22-11e8-8241-724b0cbe14e6.png">